### PR TITLE
More unit tests

### PR DESCRIPTION
**Pull request by mbelur**

### CODE REVIEW

**Review by jeremy-moffitt — COMMENTED** [review]
At a glance these tests look ok... its not clear to me how to run them however. The README should be updated, or some temporary instructions added under the next_generation directory

File: next_generation/tests/unit/mock_pint_data.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import glob
+import os
+
+from lxml import etree
+
+def images(provider):
+    base_path = os.path.dirname(os.path.abspath(__file__))
+    data_file = glob.glob(base_path + '/./mockpintdata/' + provider + '.xml')
```
Comment (jeremy-moffitt) [inline next_generation/tests/unit/mock_pint_data.py:8]: Isnt `/./mockpintdata` the same as `/mockpintdata` ? or am I missing something

**Comment by mbelur** [comment]
> At a glance these tests look ok... its not clear to me how to run them however. The README should be updated, or some temporary instructions added under the next_generation directory

I have added a READMEmd under the tests/unit folder.

File: next_generation/tests/unit/mock_pint_data.py (hunk 1 above)
Comment (mbelur) [reply next_generation/tests/unit/mock_pint_data.py]: Updated

**Review by jeremy-moffitt — APPROVED** [review]
LGTM but you probably want another reviewer before merging. Thanks for adding the instructions to run the tests.

**Review by guangyee — APPROVED** [review]

**Pull request merged**